dias/analyzers/source_spectra_analyzer.py

```diff
@@ -0,0 +1,733 @@
+
+                    index_map = handler.create_group("index_map")
+                    index_map.create_dataset("freq", data=data.index_map["freq"][:])
+                    index_map.create_dataset("pol", data=np.string_(pols))
+                    index_map.create_dataset("time", data=data.time)
+                    index_map.create_dataset("ra", data=ra)
+                    index_map.create_dataset("ha", data=ha)
+
+                    dset = handler.create_dataset("vis", data=vis)
+                    dset.attrs["axis"] = np.array(["freq", "pol", "ha"], dtype="S")
+
+                    dset = handler.create_dataset(
+                        "weight", data=tools.invert_no_zero(var)
+                    )
+                    dset.attrs["axis"] = np.array(["freq", "pol", "ha"], dtype="S")
+
+                    dset = handler.create_dataset("count", data=counter.astype(np.int))
+                    dset.attrs["axis"] = np.array(["freq", "pol", "ha"], dtype="S")
+
+                    if self.perform_fit:
+
+                        index_map.create_dataset("param", data="model_fitting")
+
+                        dset = handler.create_dataset("residual_noise", data=resid_rms)
+                        dset.attrs["axis"] = np.array(["freq", "pol"], dtype="S")
+
+                        dset = handler.create_dataset("parameter", data=parameter)
+                        dset.attrs["axis"] = np.array(
+                            ["freq", "pol", "param"], dtype="S"
+                        )
+
+                        dset = handler.create_dataset(
+                            "parameter_cov", data=parameter_cov
+                        )
+                        dset.attrs["axis"] = np.array(
+                            ["freq", "pol", "param", "param"], dtype="S"
+                        )
+
+                    handler.attrs["source"] = src
+                    handler.attrs["csd"] = csd
+                    handler.attrs["instrument_name"] = self.correlator
+                    handler.attrs["collection_server"] = subprocess.check_output(
+                        ["hostname"]
+                    ).strip()
+                    handler.attrs["system_user"] = subprocess.check_output(
+                        ["id", "-u", "-n"]
+                    ).strip()
+                    handler.attrs["git_version_tag"] = dias_version_tag
+
+                self.logger.info("File for {} successfully written out.".format(src))
+                self.run_metric.labels(source=src).inc()
+
+        if err_msg:
+            raise exception.DiasDataError(err_msg)
+
+    def get_baselines(self, indexmap, inputmap):
+        """Return baseline indices for averaging."""
+        prod = defaultdict(list)
+        prodmap = defaultdict(list)
+        dist = defaultdict(list)
+        conj = defaultdict(list)
+        cyl = defaultdict(list)
+        scale = defaultdict(list)
+
+        # Compute feed positions without rotation
```

Comment:
without --> with

Reply:
Related to this, what is the current best estimate of the rotation for default?

Reply:
The current best estimate is -0.071 deg.  We recently updated `ch_util` to use this value _by default_, so it is no longer necessary to manually set the rotation in this analyzer, you can let the `ch_util.tools.get_feed_positions` do it for you.

Reply:
Okay, will do that